packages/backend/src/config/aiConfigSchema.ts

```diff
@@ -1,11 +1,10 @@
 import { z } from 'zod';
 
-export const DEFAULT_OPENAI_MODEL_NAME = 'gpt-4.1-2025-04-14';
-export const DEFAULT_ANTHROPIC_MODEL_NAME = 'claude-sonnet-4-5-20250929';
+export const DEFAULT_OPENAI_MODEL_NAME = 'gpt-4.1';
+export const DEFAULT_ANTHROPIC_MODEL_NAME = 'claude-sonnet-4-5';
```

Comment:
this is the format we expect from the env vars, but I've also added a fallback, so in cacse user provides a snapshot version, it will still work (assuming we support that snapshot)